src/reducers/sessionReducer.js

```diff
@@ -10,6 +16,10 @@ const handleLoginSuccess = (state, { payload }) => {
   state.user = payload;
 };
 
+const handleFacebookLoginSuccess = (state, { payload }) => {
+  state.user = payload;
```

Comment:
In this case, you are only saving an email on state.user. Are you using some other props of state.user on the application? Keep this in mind in case you do

Reply:
Yes! Right now I just needed the email to show the logged user on the main page. I believe I'll need to change this in the future!